jobs/transforms/base_model_audits.py

```diff
@@ -44,6 +44,10 @@
 BASE_MODEL_ID_PATTERN = r'^[A-Za-z0-9-_]{1,%s}$' % base_models.ID_LENGTH
 MAX_CLOCK_SKEW_SECS = datetime.timedelta(seconds=1)
 
+VALIDATION_MODE_NEUTRAL = 'neutral'
```

Comment:
Please use `utils.create_enum` for these.

Reply:
Done.